docs/how-to.md

```diff
@@ -19,3 +19,21 @@ This setting should always have value greater than or equal to 1
 The `memory_per_job_gb` is a package setting that allows to scale parallel jobs by available virtual memory without swap. The default value is set to 1.0 which indicates that each parallel job requires 1 GB virtual memory
 Example: `memory_per_job_gb = 0.5` indicates that each parallel job requires 512 MB virtual memory
 This setting should always have value greater than or equal to 0.1
+
+## Enable Repeatable Builds
+
+There are times we need to ensure that the bootstrap process does not change the packages to the latest versions on its own.
+Instead, we want to use the last stable bootstrap as the only source of truth to ensure that the build process is fully
+repeatable and predictable. Moreover, we also want to allow upgrading of specific packages to rebuild new RC blockers, do bug fixes, etc.
+
+Fromager supports repeatable builds to ensure bootstrapping the versions of packages that the branch was able to build initially when it was
```

Comment:
This paragraph is a little too much focused on what we're doing downstream (mentioning branches, for example). See if you can update it to just talk about the feature in fromager, without talking about how we use it downstream.

Reply:
Done